### ISSUE TITLE
' = '  does not equal  '=' as the docs suggest

### ISSUE THREAD

**Issue by ajoslin103** [MarcScheib/aurelia-environment#2]
in your usage doc you imply that the below assignments are all roughly equivalent

but your parser doesn't strip the spaces before and after the '='

TEST1 = value

results in this: 'TEST1 '=' value'
so that window.env.TEST1 will be undefined
while window.env['TEST1 '] will equal not 'value' but ' value'

```
Assignment

To assign values, the syntax is key = value. However, all of the assignments below are valid:

TEST1 = value
TEST2= VALUE
TEST3 =VALUE
TEST4=VALUE
 TEST5 = VALUE
TEST6  =   VALUE
```

**Comment by MarcScheib** [comment]
Hi @ajoslin103 , 
this is definitively a bug. Thanks for your issue. It seems like I am lacking the test cases for this. I will get that fixed.

**Comment by MarcScheib** [comment]
Fixed in 0.1.7
